lib/wasi/src/os/console/mod.rs

```diff
@@ -0,0 +1,277 @@
+#![allow(unused_imports)]
+#![allow(dead_code)]
+
+pub mod cconst;
+
+use std::{
+    collections::HashMap,
+    io::Write,
+    ops::{Deref, DerefMut},
+    path::Path,
+    sync::{atomic::AtomicBool, Arc, Mutex},
+};
+
+use derivative::*;
+use linked_hash_set::LinkedHashSet;
+use tokio::sync::{mpsc, RwLock};
+#[allow(unused_imports, dead_code)]
+use tracing::{debug, error, info, trace, warn};
+#[cfg(feature = "sys")]
+use wasmer::Engine;
+use wasmer_vbus::{BusSpawnedProcess, SpawnOptionsConfig, VirtualBusError};
+use wasmer_vfs::{FileSystem, RootFileSystemBuilder, SpecialFile, WasiPipe};
+use wasmer_wasi_types::{types::__WASI_STDIN_FILENO, wasi::BusErrno};
+
+use super::{cconst::ConsoleConst, common::*};
+use crate::{
+    bin_factory::{spawn_exec, BinFactory, ModuleCache},
+    os::task::{control_plane::WasiControlPlane, process::WasiProcess},
+    runtime::{RuntimeStderr, RuntimeStdout},
+    state::Capabilities,
+    TtyFile, VirtualTaskManagerExt, WasiEnv, WasiRuntimeImplementation, WasiState,
+};
+
+//pub const DEFAULT_BOOT_WEBC: &'static str = "sharrattj/bash";
+pub const DEFAULT_BOOT_WEBC: &'static str = "sharrattj/dash";
+//pub const DEFAULT_BOOT_USES: [&'static str; 2] = [ "sharrattj/coreutils", "sharrattj/catsay" ];
+pub const DEFAULT_BOOT_USES: [&'static str; 0] = [];
```

Comment:
We should have no ties into specific packages here.
It points that perhaps we are not doing properly a separation of concerns here

Reply:
Removed the default, now has to be supplied as an argument.